wiki/Guides/Tablet_Purchase/en.md

```diff
@@ -1,54 +1,74 @@
+
+![](http://puu.sh/dlQH "Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)")
+
+![](http://puu.sh/dlQi "Third generation Bamboo (Connect, Capture, Create - Connect shown)")
+
+You'll want to stay away from either of the Bamboo Touch tablets, they do not feature pen input, they're just multitouch trackpads for your computer. Frankly, [Apple did it better](http://www.apple.com/magictrackpad/).
+
+The other Bamboo models come in two sizes, basically small and large. For the second generation, Bamboo Touch, Pen, and Pen and Touch were the "small" ones, whereas the Fun and Craft were the "large" ones. For the third generation, the Connect and Capture are the "small", whereas the Create is the "large". For osu!, you'll probably be looking at getting one of the "small" sized tablets because it's a common strategy to reduce play area so that you don't have to move your arm when you play. However, it's personal preference as to whether or not you want a larger tablet area, and it's especially worth considering the "large" size if you are also into digital art; drawing is much more comfortable on a larger tablet.
 
--   [Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)](http://puu.sh/dlQH)
--   [Third generation Bamboo (Connect, Capture, Create - Connect shown)](http://puu.sh/dlQi)
+Lastly, Intuos3 tablets can be had for very cheap on second-hand sites like eBay or Craigslist, and are definitely worth checking out if you're also going to be doing digital art as well. For osu!, however, the differences between the Intuos and Bamboo lines are negligible, the only real difference is that the Intuos lines have a much wider range of sizes than is available for Bamboo.
 
-You'll want to stay away from either of the Bamboo Touch tablets - they do not feature pen input, they're just multitouch trackpads for your computer. Frankly, [Apple did it better](http://www.apple.com/magictrackpad/).
-The other Bamboo models come in two sizes - basically small and large. For the second generation, Bamboo Touch, Pen, and Pen and Touch were the "small" ones, whereas the Fun and Craft were the "large" ones. For the third generation, the Connect and Capture are the "small", whereas the Create is the "large". For osu!, you'll probably be looking at getting one of the "small" sized tablets because it's a common strategy to reduce play area so that you don't have to move your arm when you play. However, it's personal preference as to whether or not you want a larger tablet area, and it's especially worth considering the "large" size if you are also into digital art; drawing is much more comfortable on a larger tablet.
+tl;dr Wacom's the recommend brand. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used Intuos3 if you're also into art.
```

Comment:
tl;dr -> "In short, "

probably rephrase to "Wacom is the recommended brand from most serious osu! players"